samples/matter/contact_sensor/prj.conf

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable CHIP
+CONFIG_CHIP=y
+CONFIG_CHIP_PROJECT_CONFIG="src/chip_project_config.h"
+
+# Configure ZAP file name
+CONFIG_NCS_SAMPLE_MATTER_ZAP_FILE_PATH="${APPLICATION_CONFIG_DIR}/src/default_zap/contact_sensor.zap"
+
+# 32768 == 0x8000 (example Product ID added temporaly,
+# as there is no assigned PID on the following list:
+# https://github.com/project-chip/connectedhomeip/blob/482e6fd03196a6de45465a90003947ef4b86e0b1/docs/examples/discussion/PID_allocation_for_example_apps.md)
```

Comment:
Could you please check if we have a PID for the contact sensor?

Reply:
I'm sorry, but I'm new in this project and I don't know where can I find this information. Would you mind pointing me to the proper resources?